plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -74,31 +69,24 @@
 public abstract class AbstractTrinoCatalog
         implements TrinoCatalog
 {
-    // Be compatible with views defined by the Hive connector, which can be useful under certain conditions.
+    public static final String TRINO_CREATED_BY_VALUE = "Trino Iceberg connector";
```

Comment:
This constant can stay `protected` (as it was before this PR).

Reply:
It is used in `TrinoHiveCatalogFactory`.